### PR TITLE
Improve repo README text

### PR DESCRIPTION
**Pull request by vahram-papazyan**
No changelog

This PR includes text improvements in the repo README

### CODE REVIEW

**Comment by StachuDotNet** [comment]
Hey there, thanks for this work!

The text changes here all look good to me. That said, the icons I find to be a bit distracting. Would you mind removing those? Including the `▶` - that feels like I should be able to click it to expand something.

**Comment by pbiggar** [comment]
I'm a fan of the icons - I feel this is the direction most projects/docs are going these days and it adds character and clarity.

**Comment by StachuDotNet** [comment]
Yeah maybe I'm just old-fashioned :) Happy to be the exception.
Could we please at least remove that play button? I can't help but want to click it.

**Comment by StachuDotNet** [comment]
yeah, the rocket there is definitely better :)

**Comment by StachuDotNet** [comment]
I think it's a good separation - editing content is a good thing to do, before trying to add new content. It's an easier task, so good to have it be a separate, "complete-able" section in the doc.

File: README.md [hunk 1]
```diff
@@ -41,96 +39,96 @@ npm install
...
+### 🔵 Tutorials
 
-- "teaching a child how to cook"
-- tells them what to do
-- does not explain why they are doing it
-- include specifics and let them learn the generalities over time
-- assume that "obvious" things are not known
-- do not include choices
-- should be bulletproof
+- A tutorial "teaches a child how to cook."
```
Comment (StachuDotNet) [inline README.md:63]:
```suggestion
A tutorial that teaches a child how to cook

```

File: README.md [hunk 2]
```diff
@@ -41,96 +39,96 @@ npm install
...
+- Does not include choices.
+- Should be bulletproof.
 
-### How-to guides
+### 🔵 How-to guides
 
-- "recipes"
-- should have very specific names such as "how to add a custom domain to Darklang"
+- "Recipes" showing how to do something.
+- Should have very specific names, such as "How to add a custom domain to Darklang."
```
Comment (StachuDotNet) [inline README.md:74]:
```suggestion
- "Recipes" showing how to do something should have very specific names, such as "How to add a custom domain to Darklang."
```

**Review by StachuDotNet — APPROVED** [review]
Great improvements - thank you!

**Pull request merged**